docs/application/dotnet/guides/tizen.circularUI/CircleStepper.md

```diff
@@ -0,0 +1,109 @@
+---
+uid: Tizen.Wearable.CircularUI.doc.CircleStepper
+summary: CircleStepper control guide
+---
+# CircleStepper
+
+`CircleStepper` allows the user to select a value from a range of incremental values specified with the `Minimum`, `Maximum`, and `Increment` properties.
+It is an extension of [Xamarin.Forms.Stepper](https://developer.xamarin.com/api/type/Xamarin.Forms.Stepper/).
+MarkerColor, MarkerLineWidth, and LabelFormat have been added to the [Xamarin.Forms.Stepper](https://developer.xamarin.com/api/type/Xamarin.Forms.Stepper/).
+You can change value with [Bezel interaction](https://developer.tizen.org/design/wearable/interaction/bezel-interactions).
+In order to receive [Rotary event](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/event-handling#rotary), it must be registered as `RotaryFocusObject` property of [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage).
+
+![](data/CircleStepper_property.png)
+
+**WARNING: [CircleListView](xref:Tizen.Wearable.CircularUI.doc.CircleListView), [CircleDateTimeSelector](xref:Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector), [CircleScrollView](xref:Tizen.Wearable.CircularUI.doc.CircleScrollView), [CircleStepper](xref:Tizen.Wearable.CircularUI.doc.CircleStepper) must be contained by `CirclePage` or [CircleSurfaceEffectBehavior](xref:Tizen.Wearable.CircularUI.doc.CircleSurfaceEffectBehavior) should be added in [Behaviors](https://developer.xamarin.com/api/type/Xamarin.Forms.Behavior/) of [Page](https://developer.xamarin.com/api/type/Xamarin.Forms.Page/) that contains these controls. If other `pages` contain these controls, it may cause exception or can't display control.**
+
+## Adding CircleStepper at CirclePage
```

Comment:
In other guides, the title of the sub paragraph starts with "Adding".
https://developer.tizen.org/development/guides/native-application/user-interface/efl/ui-components/wearable-ui-components/circle-spinner

new sentence:
Adding CircleStepper in CirclePage

Reply:
@hj-na-park according to our checklist we avoid using gerund (ing) in the heading.
The main reason to avoid usage of gerund in the sentence is that during translation the meaning of the word will change.

Reply:
OK. I'll change 'Adding' to 'Add'.